ui/src/notebooks/components/header/Submit.tsx

```diff
@@ -17,6 +18,7 @@ const COMMENT_REMOVER = /(\/\*([\s\S]*?)\*\/)|(\/\/(.*)$)/gm
 
 export const Submit: FC = () => {
   const {query} = useContext(QueryContext)
+  const {onNotify, setQueryToLoading} = useContext(SubmitQueryButtonContext)
```

Comment:
this context doesn't have a provider. what are you trying to do in here?

Reply:
Basically trying to mock out the functionality that's introduced in wrapping the component to redux. Was a bit confused by what I was actually doing since I just planned to use it as a way of satisfying the linter / following what was happening here